CMakeLists.txt

```diff
@@ -8,6 +8,15 @@ project("Libmultiprocess" CXX)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED YES)
 
+option(MULTIPROCESS_RUN_CLANG_TIDY "Run clang-tidy with the compiler." OFF)
```

Comment:
In commit "Add option to run `clang-tidy` with compiler" (f3b2bfcbe7ecca087d75e96df1496687f8ff4c0a)

Would it make sense to call the option `ENABLE_CLANG_TIDY` instead of `MULTIPROCESS_RUN_CLANG_TIDY`? I don't see a reason to prefix option names with the project name, and the other options don't do this

Reply:
The goal of prefixing is to minimize risk if option name clash when this library is a subproject of another project. For example, https://github.com/bitcoin-core/secp256k1/pull/1113.

Reply:
Ok, I didn't think about this being used as a subdirectory inside another project. Could we then prefix project options with the project name like `Libmultiprocess_ENABLE_CLANG_TIDY`. I do also prefer `ENABLE` to `RUN` for the cmake option name, because at the `CMakeLists.txt` level, this isn't adding more build steps, just setting another option.

Reply:
Thanks! [Updated](https://github.com/chaincodelabs/libmultiprocess/pull/83#issuecomment-1430948349).